go-update/src/main/java/org/reactome/release/goupdate/GoTermInstanceModifier.java

```diff
@@ -282,6 +282,22 @@ public static boolean isGoTermDeleteable(GKInstance instance) throws Exception
 	}
 	
 	
+	public void deleteSecondaryGOInstance(GKInstance primaryGOTerm, StringBuffer deletionStringBuffer)
```

Comment:
I think StringBuilder is preferred to StringBuffer now for single threads.

https://stackoverflow.com/questions/355089/difference-between-stringbuilder-and-stringbuffer

Reply:
I probably used the thread-safe structure in case I figured out a way to parallelize that bit of code, or what was calling it. Or there may have been a block of code that *did* execute on multiple threads and also referenced `GoTermsUpdater.deletionStringBuilder`, which is why it is a `StringBuffer` and not a `StringBuilder`.